### PR TITLE
[GT-172] Add blank line across the codebase for PHP files

### PR DESCRIPTION
**Pull request by Sae126V**
Adding a single empty Line across the codebase at the EOF. There won't be any impact in-terms of functionality or behaviour. This change applies ONLY to the codebase.

"Please feel free to give any thoughts or comments on this" ---

"Resolves [GT-172]"

[GT-172]: https://stfc.atlassian.net/browse/GT-172?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: htdocs/web_portal/views/downtime/edit_downtime.php [hunk 1]
```diff
@@ -225,7 +225,7 @@ class="form-control" id="Select_Sites" name="select_sites" size="10"
         $('#startDate').data("DateTimePicker").date("<?php echo date_format($startDate,"d/m/Y"); ?>");
         $('#endDate').data("DateTimePicker").date("<?php echo date_format($endDate,"d/m/Y"); ?>");
 
-        //Set the start and finish times (don't echo in the full date, just the time values, time widget didn�t like timestamp with date)
+        //Set the start and finish times (don't echo in the full date, just the time values, time widget didn�t like timestamp with date)
```
Comment (rowan04) [inline htdocs/web_portal/views/downtime/edit_downtime.php:228]: ??

```suggestion
        //Set the start and finish times (don't echo in the full date, just the time values, time widget didn't like timestamp with date)
```

Comment (Sae126V) [reply htdocs/web_portal/views/downtime/edit_downtime.php:228]: Just to add on. The line you are seeing is only visible in the git(Browser - JS engine seems hard time understanding the characters). But this is not part of the commit.

Comment (gregcorbett) [reply htdocs/web_portal/views/downtime/edit_downtime.php:228]: can this change be omitted from this PR?

**Review by gregcorbett — CHANGES_REQUESTED** [review]
LGTM, just a minor thing we a strange character.

File: htdocs/web_portal/views/downtime/edit_downtime.php (hunk 1 above)
Comment (Sae126V) [reply htdocs/web_portal/views/downtime/edit_downtime.php]: Removed the strange/unidentified character from the PR.

Comment (gregcorbett) [reply htdocs/web_portal/views/downtime/edit_downtime.php]: ah sorry, I mean't remove the change to this line - so this PR only deals with whitespace/newlines at the end of files. Though please do stick [40942ad](https://github.com/GOCDB/gocdb/pull/457/commits/40942ad73e8089b05c3262974080df1b25da0f0b) in a seperate PR 😄 .

Comment (Sae126V) [reply htdocs/web_portal/views/downtime/edit_downtime.php]: @gregcorbett , If you see my initial commit 40e8919cc409e9e8fe1456966e2ca7109bc0dd56. "It doesn't have that change at all".  But GIT or GitHub (Not sure) somehow picked up that line saying their is a strange character.
 Even If I remove that change GIT will complain again regarding that character.

Comment (gregcorbett) [reply htdocs/web_portal/views/downtime/edit_downtime.php]: If Gits detecting it as a change, you should be able to not commit that change? either way - putting [40942ad](https://github.com/GOCDB/gocdb/pull/457/commits/40942ad73e8089b05c3262974080df1b25da0f0b) into a separate PR will remove the change from this PR.

Comment (Sae126V) [reply htdocs/web_portal/views/downtime/edit_downtime.php]: I am attaching the history of "https://github.com/GOCDB/gocdb/commit/40e8919cc409e9e8fe1456966e2ca7109bc0dd56" vs previous commit that showing in red lines of what I have changed.

edit_downtime

Comment (Sae126V) [reply htdocs/web_portal/views/downtime/edit_downtime.php]: No @gregcorbett . GIT is NOT showing me that change expect in the browser. I don't see that change when I am doing commit.

Comment (gregcorbett) [reply htdocs/web_portal/views/downtime/edit_downtime.php]: how weird, it shows up for me in Git as a change. Regardless, putting [40942ad](https://github.com/GOCDB/gocdb/pull/457/commits/40942ad73e8089b05c3262974080df1b25da0f0b) into a separate PR will remove the change from this PR.

Comment (Sae126V) [reply htdocs/web_portal/views/downtime/edit_downtime.php:228]: Deleted the change from this PR @gregcorbett 👍

Comment (Sae126V) [reply htdocs/web_portal/views/downtime/edit_downtime.php:228]: This is being addressed as part of https://github.com/GOCDB/gocdb/pull/471.

**Comment by gregcorbett** [comment]
Can you rebase this ontop of the latest dev? That should remove the strange character change, as that has been merged in (see #471) separately.

**Comment by gregcorbett** [comment]
Looks like you'll need to update your local copy of the upstream dev branch and rebase again - there's still conflicts.

**Comment by gregcorbett** [comment]
I've documented the fix to the weirdness here: https://stfc.atlassian.net/wiki/spaces/GTT/pages/160170035/GOCDB+Developement#Troubleshooting

**Comment by Sae126V** [comment]
Rebased on dev.

Verified

**Comment by Sae126V** [comment]
Rebased and verified.

**Review by gregcorbett — APPROVED** [review]
Looks good to me :)

**Pull request merged**